src/main/scala/mesosphere/marathon/ZookeeperConf.scala

```diff
@@ -16,16 +16,16 @@ trait ZookeeperConf extends ScallopConf {
   private val zkURLPattern = s"""^zk://(?:$userAndPass@)?($hostAndPort(?:,$hostAndPort)*)(/$zkNode(?:/$zkNode)*)$$""".r
 
   lazy val zooKeeperTimeout = opt[Long]("zk_timeout",
-    descr = "The timeout for ZooKeeper in milliseconds",
+    descr = "The timeout for Zookeeper in milliseconds.",
     default = Some(10 * 1000L)) //10 seconds
 
   lazy val zooKeeperSessionTimeout = opt[Long]("zk_session_timeout",
-    descr = "The timeout for zookeeper sessions in milliseconds",
+    descr = "The timeout for Zookeeper sessions in milliseconds",
     default = Some(10 * 1000L) //10 seconds
   )
 
   lazy val zooKeeperUrl = opt[String]("zk",
-    descr = "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
+    descr = "Zookeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
```

Comment:
On Fri, Nov 13, 2015 at 11:05 AM, Matthias Veit <notifications@github.com>
wrote:

> In src/main/scala/mesosphere/marathon/ZookeeperConf.scala
> <https://github.com/mesosphere/marathon/pull/2630#discussion_r44766767>:
>
> >      default = Some(10 * 1000L) //10 seconds
> >    )
> >
> >    lazy val zooKeeperUrl = opt[String]("zk",
> > -    descr = "ZooKeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
> > +    descr = "Zookeeper URL for storing state. Format: zk://host1:port1,host2:port2,.../path",
>
> I replaced all ZooKeeper with Zookeeper as suggested...
>

I took a look at https://zookeeper.apache.org/ before writing those
comments, the official spelling is: "ZooKeeper".

At least now we spell it consistently in all the descriptions. I wouldn't
make it a blocker to switch to the official spelling.

Thanks,

-Gastón